units/util.go

```diff
@@ -29,24 +30,30 @@ func HandlePoisonedHost(ctx context.Context, env evergreen.Environment, h *host.
 			}
 
 			for _, container := range containers {
-				catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, container, reason))
+				catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, &container, reason))
 			}
-			catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, *parent, reason))
+			catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, parent, reason))
 		}
 	}
-	catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, *h, reason))
+	catcher.Add(DisableAndNotifyPoisonedHost(ctx, env, h, reason))
 	return catcher.Resolve()
 }
 
-func DisableAndNotifyPoisonedHost(ctx context.Context, env evergreen.Environment, h host.Host, reason string) error {
+func DisableAndNotifyPoisonedHost(ctx context.Context, env evergreen.Environment, h *host.Host, reason string) error {
 	if h.Status == evergreen.HostDecommissioned || h.Status == evergreen.HostTerminated {
```

Comment:
Can you update this to also check for quarantined?

Reply:
done